components/Card.js

```diff
@@ -15,7 +15,8 @@ export default function Card({ showTranslation, translation, content }) {
 
 const StyledCard = styled.div`
   background-color: #fcfcff;
-  border: 1px solid #d4d4d4;
+  box-shadow: 0px 0px 25px 6px rgba(193, 193, 209, 0.52);
```

Comment:
Thank you, Federico! Yes... I didn't question it and let it like this because it's from a website, CJ recommended for calculating the box-shadow :) But yes, there is a new convention, you're right.